docs/_source/community/developer_docs.md

```diff
@@ -38,7 +38,7 @@ git remote add upstream https://github.com/argilla-io/argilla.git
 
 ### Install dependencies
 
-To be able to use Argilla via either the Python client or the Python Command Line Interface (CLI), you'll need to 
+To be able to use Argilla via either the Python client or the Python Command Line Interface (CLI), you'll need to
 install `argilla` and its extra dependencies, if applicable. To do so, you can either install it via `pip` or setup
```

Comment:
should we mention the extras specifically?

Reply:
What do you mean with that? Anyway the change on this file should be tackled in the parent PR not in this once, as that was automatically fixed by the pre-commit bot without me noticing :( It was just the last space removal BTW